### ISSUE TITLE
Status?

### ISSUE THREAD

**Issue by dandv** [ephemerisle/ephemerisle.github.io#4]
Is this repo abandoned in favor of http://ephemerisle.org ?

**Comment by cypof** [comment]
Yes people find it easier to use a regular wiki, so Adam and several others
worked on switching.
